Gemfile

```diff
@@ -18,6 +18,8 @@
 source 'https://rubygems.org'
 gemspec
 
+gem 'redcarpet', require: false
```

Comment:
Does this even need to be in the file? Same with `yard` too? I make this suggestion as you manually install them in the Github Action, and you don't actually need them to build or run the tests. This would fix the jruby issue too.

Reply:
Great idea